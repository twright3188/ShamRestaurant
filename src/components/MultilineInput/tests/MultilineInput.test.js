import React from 'react';
import renderer from 'react-test-renderer';
import { MultilineInput } from '../index';

it('multiline input render', () => {
    const tree = renderer.create(<MultilineInput />).toJSON();
    expect(tree).toMatchSnapshot();
});
