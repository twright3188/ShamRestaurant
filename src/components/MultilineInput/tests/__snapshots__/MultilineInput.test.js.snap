// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`multiline input render 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "borderBottomColor": "rgba(34, 38, 36, 0.15)",
        "borderBottomWidth": 1,
        "flexWrap": "wrap",
        "paddingHorizontal": 15,
        "paddingVertical": 14,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "flexWrap": "wrap",
            "fontFamily": "Helvetica",
            "fontWeight": "400",
          },
          Object {
            "color": "#222624",
            "fontSize": 16,
            "lineHeight": 24,
          },
          Object {},
        ]
      }
    >
      
    </Text>
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#399AE5",
            "fontSize": 24,
          },
          undefined,
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ï†Ÿ
    </Text>
  </View>
  <View
    style={
      Object {
        "flexWrap": "wrap",
        "paddingRight": 32,
        "width": "100%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoFocus={false}
      multiline={true}
      onChangeText={null}
      onKeyPress={[Function]}
      placeholder=""
      rejectResponderTermination={true}
      returnKeyType="done"
      style={
        Object {
          "color": "#222624",
          "fontSize": 14,
          "width": "100%",
        }
      }
      textAlignVertical="top"
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
</View>
`;
